go.mod

```diff
@@ -19,6 +19,7 @@ require (
 	github.com/google/gofuzz v1.1.0 // indirect
 	github.com/leodido/go-urn v1.2.1 // indirect
 	github.com/pmezard/go-difflib v1.0.0 // indirect
+	github.com/serverlessworkflow/sdk-go v1.0.0 // indirect
```

Comment:
This is the old version of the SDK. Why it was added here? Can you review the imports?

Reply:
I honestly don't know why. probably something with my ide.